### PR TITLE
Hotfix: not following up

### PR DESCRIPTION
**Pull request by wannacfuture**
Need to be merged ASAP

### CODE REVIEW

File: src/helpers/issue.ts [hunk 1]
```diff
@@ -67,6 +67,24 @@ export const listIssuesForRepo = async (state: "open" | "closed" | "all" = "open
   }
 };
 
+export const listAllIssuesForRepo = async (state: "open" | "closed" | "all" = "open") => {
+  const issuesArr = [];
+  let fetchDone = false;
+  const perPage = 30;
```
Comment (pavlovcik) [inline src/helpers/issue.ts:73]: 30 seems unusual is this confirmed?

I would maybe expect 25 due to the notifications view paginating every 25.

**Review by pavlovcik — COMMENTED** [review]
What about rate limiting?

File: src/handlers/wildcard/unassign.ts [hunk 2]
```diff
@@ -102,7 +102,7 @@ const lastActivityTime = async (issue: Issue): Promise<Date> => {
       .sort((a, b) => new Date(b.commit.committer?.date ?? 0).getTime() - new Date(a.commit.committer?.date ?? 0).getTime());
     const prComments = (await getCommentsOfIssue(pr.number))
       .filter((comment) => comment.user.login === assignees[0])
-      .sort((a, b) => new Date(a.created_at).getTime() - new Date(b.created_at).getTime());
+      .sort((a, b) => new Date(b.created_at).getTime() - new Date(a.created_at).getTime());
```
Comment (pavlovcik) [inline src/handlers/wildcard/unassign.ts:105]: Why was this swapped?

File: src/helpers/issue.ts (hunk 1 above)
Comment (wannacfuture) [reply src/helpers/issue.ts]: sure, fixed.

File: src/handlers/wildcard/unassign.ts (hunk 2 above)
Comment (wannacfuture) [reply src/handlers/wildcard/unassign.ts:105]: Yeah, the previous one was wrong.
We need to get the latest comment on the PR but seems it was getting the first comment.

File: src/helpers/issue.ts (hunk 1 above)
Comment (wannacfuture) [reply src/helpers/issue.ts]: rate limitting added

File: src/helpers/issue.ts [hunk 3]
```diff
@@ -60,13 +60,33 @@ export const listIssuesForRepo = async (state: "open" | "closed" | "all" = "open
...
+  let fetchDone = false;
+  const perPage = 25;
+  let curPage = 1;
+  while (!fetchDone) {
+    const issues = await listIssuesForRepo(state, perPage, curPage);
+
+    // push the objects to array
+    issuesArr.push(...issues);
+
+    if (issues.length < perPage) fetchDone = true;
```
Comment (pavlovcik) [inline src/helpers/issue.ts:83]: In your testing can you tell me what the value of `issues.length` is on a full pagination?

File: src/helpers/issue.ts (hunk 1 above)
Comment (wannacfuture) [reply src/helpers/issue.ts]: It doesn't matter actually, we can set any random number for that.

**Pull request closed without merging**